src/start/RealmScreen.js

```diff
@@ -17,7 +17,7 @@ type SelectorProps = {|
 
 type Props = $ReadOnly<{|
   navigation: NavigationScreenProp<{
-    params: ?{|
+    params: {|
```

Comment:
e1922546c RealmScreen: Assume `navigation.state.params` will exist.

Oops! This commit is wrong; please don't merge until I've pushed a fix. I forgot about reaching RealmScreen through `navReducer`.

Reply:
OK, done.